From 674023a6180033c5651128032a39d25b2171cd68 Mon Sep 17 00:00:00 2001
From: Mashopy <eliasgheeraert@gmail.com>
Date: Mon, 12 Oct 2026 19:19:10 +0000
Subject: [PATCH] fastbootd: Always return false for GetDeviceLockStatus()

fenrir lks can still flash stuff through bootloader and fastboot

Rebased onto lineage-24.0, where GetDeviceLockStatus() compares
against "orange" instead of "green".
Original: Nothing-2A/android_system_core@0d5990a

Change-Id: Ica59dab533555af1d12c62b9cc71de55d5b51146
---
 fastboot/device/utility.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fastboot/device/utility.cpp b/fastboot/device/utility.cpp
index e12ee64..0652d93 100644
--- a/fastboot/device/utility.cpp
+++ b/fastboot/device/utility.cpp
@@ -195,7 +195,7 @@ std::vector<std::string> ListPartitions(FastbootDevice* device) {
 }
 
 bool GetDeviceLockStatus() {
-    return android::base::GetProperty("ro.boot.verifiedbootstate", "") != "orange";
+    return false;
 }
 
 bool UpdateAllPartitionMetadata(FastbootDevice* device, const std::string& super_name,
-- 
2.43.0

